CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [ENHANCEMENT] Updated Prometheus to latest. Includes changes from prometheus#9239, adding 15 new functions. Multiple TSDB bugfixes prometheus#9438 & prometheus#9381. #4524
 * [ENHANCEMENT] Query Frontend: Add setting `-frontend.forward-headers-list` in frontend  to configure the set of headers from the requests to be forwarded to downstream requests. #4486
 * [ENHANCEMENT] Blocks storage: Add `-blocks-storage.azure.http.*`, `-alertmanager-storage.azure.http.*`, and `-ruler-storage.azure.http.*` to configure the Azure storage client. #4581
+* [ENHANCEMENT] Add length and limit to labelNameTooLongError and labelValueTooLongError #4595
```

Comment:
Nit: CHANGELOG is for admins, not for developers, so it is better to use English than type names.

Reply:
Oh, I see. How about:
`Add length and limit to error message when label name and value are too long`

Reply:
Great!

Reply:
Done.